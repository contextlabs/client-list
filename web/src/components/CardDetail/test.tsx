import { render, screen } from '@testing-library/react';

import { CardDetail } from '.';

const noop = () => {};

describe('<CardDetail />', () => {
  const props = {
    name: 'CardDetail Name',
    title: 'CardDetail Title',
    avatar: 'https://gravatar.com/4405735f6f3129e0286d9d43e7b460d0',
    nationality: 'Global',
    quote: 'Hello'
  };

  it('should render the CardDetail as default', () => {
    const { container } = render(<CardDetail {...props} />);
    expect(container.firstChild).toMatchSnapshot();

    expect(screen.getByRole('heading', { name: /CardDetail Title/i })).toBeInTheDocument();
    expect(screen.getByRole('heading', { name: /CardDetail Name/i })).toBeInTheDocument();
  });
});
