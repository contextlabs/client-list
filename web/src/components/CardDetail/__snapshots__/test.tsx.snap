// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardDetail /> should render the CardDetail as default 1`] = `
<div
  class="w-32 mx-auto relative -mt-24 mb-6"
>
  <img
    class="inline-block rounded-full w-32 h-32"
    src="https://gravatar.com/4405735f6f3129e0286d9d43e7b460d0"
  />
</div>
`;
